src/QsCompiler/SyntaxProcessor/TypeInference/Diagnostic.fs

```diff
@@ -36,19 +36,17 @@ type Diagnostic =
     | CompilerDiagnostic of QsCompilerDiagnostic
 
 module Diagnostic =
-    let withParents expected actual diagnostic =
-        let hasSameRange (type1: ResolvedType) : ResolvedType option -> _ =
```

Comment:
Was `hasSameRange` publicly accessible? If so, should we consider deprecating it now and removing it later in case there's something relying on it?

Reply:
`hasSameRange` was a local variable inside the `withParents` function. (Everything in this file is internal, also - see [Diagnostic.fsi](https://github.com/microsoft/qsharp-compiler/blob/samarsha/fix-call-diagnostic-ranges/src/QsCompiler/SyntaxProcessor/TypeInference/Diagnostic.fsi).)